docs/install.md

```diff
@@ -230,6 +230,24 @@ SHARP_IGNORE_GLOBAL_LIBVIPS=1 npm install --arch=x64 --platform=linux --libc=gli
 To get the best performance select the largest memory available.
 A 1536 MB function provides ~12x more CPU time than a 128 MB function.
 
+### serverless-esbuild
+
+To deploy AWS Lambda using serverless framework, esbuild and serverless-esbuild on machines other than Linux x64 (glibc), first ensure sharp is excluded from bundling via the
+[exclude option](https://www.serverless.com/plugins/serverless-esbuild#options)
+configuration.
+Make changes inside `serverless.yml` under `esbuild` with the following.
+```
+esbuild:
+    # Keep existing changes
+    external:
+      - sharp
+    packagerOptions:
+      scripts:
+        - npm install --arch=x64 --platform=linux sharp
+```
+After successful deployment, comment out the scripts `- npm install --arch=x64 --platform=linux sharp` if you want to use `serverless-offline`
```

Comment:
Given this section is about deployment and not local development, do we need this sentence?

Reply:
I have kept this line just as a precaution so that one does not get stuck with `serverless-offline` if they one to test the lambdas locally.